.github/workflows/ci.yml

```diff
@@ -32,6 +32,6 @@ jobs:
               env:
                 CONFIG: ${{matrix.config}}
                 ARCH: ${{matrix.arch}}
+                COMPILER: ${{matrix.dc}}
```

Comment:
this will likely be unused, since `DC` is set as environment variable by setup-dlang (which is preferred in the shell script)